docs/src/m2md.m

```diff
@@ -1,47 +1,47 @@
-function m2md( fileIn, fileOut, includeCodeBoolean, iniLine )
+function m2md(fileIn, fileOut, includeCodeBoolean, iniLine)
 
-fidIn  = fopen( fileIn, 'r' );
-fidOut = fopen( fileOut,'w' );
+  fidIn  = fopen(fileIn, 'r');
+  fidOut = fopen(fileOut, 'w');
 
-for i=1:iniLine
-  currentLine = fgetl( fidIn ) ;
-end
+  for i = 1:iniLine
+    currentLine = fgetl(fidIn);
+  end
 
-isInCodeBlock = ~( length( currentLine )>=3 && strcmp( currentLine(1:3), '%md' ) ) ;
-lineCount = 0 ;
+  isInCodeBlock = ~(length(currentLine) >= 3 && strcmp(currentLine(1:3), '% md'));
+  lineCount = 0;
 
-while ~feof( fidIn )
+  while ~feof(fidIn)
 
-  lineCount = lineCount + 1;
-  if lineCount ~= 1
-    currentLine = fgetl( fidIn ) ;
-  end
+    lineCount = lineCount + 1;
+    if lineCount ~= 1
+      currentLine = fgetl(fidIn);
+    end
 
-  if length( currentLine )>=7 && strcmp( currentLine((end-6):end), '%hidden' )
-    % hidden line do not do anything
+    if length(currentLine) >= 7 && strcmp(currentLine((end - 6):end), '% hidden')
+      % hidden line do not do anything
 
-  elseif length( currentLine )>=3 && strcmp( currentLine(1:3), '%md' ) % not code
+    elseif length(currentLine) >= 3 && strcmp(currentLine(1:3), '% md') % not code
```

Comment:
```suggestion
    elseif length(currentLine) >= 4 && strcmp(currentLine(1:4), '% md') % not code
```

Reply:
this is one of the lines I think should be edited to have the deploy working.

Reply:
Got you, thanks for the MR. This one should be ready to go.